backend/actions/Model/getDocuments.js

```diff
@@ -50,7 +50,10 @@ module.exports = ({ db }) => async function getDocuments(params) {
     sort({ _id: -1 });
 
 
-  const schemaPaths = Model.schema.paths;
+  let schemaPaths = {};
+  for (const path of Object.keys(Model.schema.paths)) {
+    schemaPaths[path] = { instance: Model.schema.paths[path]}
```

Comment:
Make this `instance: Model.schema.paths[path].instance` so you don't need the repetitive `instance.instance` on the frontend

Reply:
I did that and the result was that there were other cases where we needed the rest of the object properties. I included my fix, let me know if you want it done differently.